pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -111,7 +111,7 @@ var _ = Describe("AccessCredentials", func() {
 		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).Return("", nil)
 
 		err := manager.agentSetUserPassword(domName, user, password)
-		Expect(err).To(BeNil())
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
This fits a direct approach: `Expect(manager.agentSetUserPassword(domName, user, password)).To(Succeed())`

Reply:
Done